perfmetrics/scripts/presubmit_test/pr_perf_test/build.sh

```diff
@@ -31,6 +31,7 @@ sudo apt-get install fio -y
 
 # Run on master branch
 cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse"
+git checkout master
```

Comment:
isn't it by default master?

Reply:
default it is master but when it will hit through PR, it is check outing to PR branch. It will switch to the latest commit id on which it is hitting.